app/assets/javascripts/components/general_button.es6.jsx

```diff
@@ -1,5 +1,10 @@
 class GeneralButton extends React.Component {
 
+  constructor(props) {
+    super(props);
```

Comment:
Looking up 'super', I see that GeneralButton is getting props from its parent. Does that mean route, action, content are from React.Component?